scripts/workloads-spin-registry-push.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I am a little confused by the title of this script.We don't seem to be doing any `spin registry push` here. This script seems more of `print-cluster-status.sh`

Reply:
this script is actually deploying the workloads created using "spin registry push" command. maybe I should rename this to `deploy-spin-registry-push-workloads.sh`. thoughts?

Reply:
👍 , I've renamed the script to `deploy-workloads.sh`, and it accepts a parameter to deploy the workload type. kindly let me know if this makes sense. happy to hear if you have any other suggestions. thank you.